snekbox/__main__.py

```diff
@@ -13,18 +13,19 @@ def parse_args() -> argparse.Namespace:
     )
     parser.add_argument("code", help="the Python code to evaluate")
     parser.add_argument(
-        "nsjail_args",
-        action="store_const",
-        const=[],
+        "--nsjail-args",
+        nargs="*",
+        default=[],
+        dest="nsjail_args",
```

Comment:
What does the help output look like now?

Reply:
Why is the default shown twice now?

Reply:
They're now options rather than positional arguments. That's confusing.

Reply:
> Why is the default shown twice now?

We hard-coded it previously but now switching it from store_const appears to display it again, we can remove our hardcoding.

> They're now options rather than positional arguments. That's confusing.

Yeah, I'll be honest I'd just updated it to maintain functionality because argparse in 3.14 broke it, hadn't looked at how it changed the help command.

I'll have a play around to see if I can maintain behaviour on 3.14 and keep the help command useful.

Reply:
Fixed this now:
```
root@038be88be536:/snekbox# snekbox 'some code' --some-nsjail-arg 1234 --- -c abcdef --some-python-arg
Namespace(code='some code', nsjail_args=['--some-nsjail-arg', '1234'], py_args=['-c', 'abcdef', '--some-python-arg'])
```

Usage is now:
```
usage: snekbox [-h] code [nsjail_args ...] [--- py_args ...]

positional arguments:
  code                  the Python code to evaluate
  nsjail_args           override configured NsJail options (default: [])
  py_args               arguments to pass to the Python process (default: ['-c'])

options:
  -h, --help            show this help message and exit
```

Duplicated defaults are also removed. It's a bit cursed but it does work.